<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="DESKTOP-K4A2ESN" name="com.toolsQA.tests.practiseFormpagetest" tests="8" failures="1" timestamp="2022-10-06T13:22:36 IST" time="4.005" errors="0">
  <testcase name="verifyFirstName" time="0.367" classname="com.toolsQA.tests.practiseFormpagetest"/>
  <testcase name="verifysubject" time="0.401" classname="com.toolsQA.tests.practiseFormpagetest"/>
  <testcase name="verifyaddress" time="1.223" classname="com.toolsQA.tests.practiseFormpagetest"/>
  <testcase name="verifyLasttName" time="0.288" classname="com.toolsQA.tests.practiseFormpagetest"/>
  <testcase name="verifygender" time="0.466" classname="com.toolsQA.tests.practiseFormpagetest"/>
  <testcase name="verifyhobbies" time="0.352" classname="com.toolsQA.tests.practiseFormpagetest"/>
  <testcase name="verifymobilenumber" time="0.166" classname="com.toolsQA.tests.practiseFormpagetest">
    <failure type="java.lang.AssertionError" message="expected [24623222] but found [&quot;24623222&quot;]">
      <![CDATA[java.lang.AssertionError: expected [24623222] but found ["24623222"]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at com.toolsQA.tests.practiseFormpagetest.verifymobilenumber(practiseFormpagetest.java:85)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- verifymobilenumber -->
  <testcase name="verifyemail" time="0.742" classname="com.toolsQA.tests.practiseFormpagetest"/>
</testsuite> <!-- com.toolsQA.tests.practiseFormpagetest -->
